.travis.yml

```diff
@@ -1,16 +1,54 @@
 language: python
+
 python:
   - 3.6
 
+os:
+  - linux
+  - linux-ppc64le
+
+env:
+  - DOCKERFILE=Dockerfile IMAGE=max-base
+  - DOCKERFILE=Dockerfile.pai IMAGE=max-base-powerai
+
+matrix:
+  exclude:
+  - env: DOCKERFILE=Dockerfile IMAGE=max-base
+    os: linux-ppc64le
+
+dist: xenial
```

Comment:
I believe we used xenial on our ppc64le builders for a newer docker version. We may be able to update docker instead, or this might not be relevant any longer. I can test this.

Reply:
`xenial` is the default as of this month: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment so no need for this line